draft-ietf-quic-transport.md

```diff
@@ -865,92 +865,85 @@ endpoint, as described in {{termination}}.
 ## Matching Packets to Connections {#packet-handling}
 
 Incoming packets are classified on receipt.  Packets can either be associated
-with an existing connection, be discarded, or - for servers - potentially create
-a new connection.
-
-Packets that can be associated with an existing connection are handled according
-to the current state of that connection.  Packets are associated with existing
-connections using connection ID if it is present; this might include connection
-IDs that were advertised using NEW_CONNECTION_ID ({{frame-new-connection-id}}).
-Packets without connection IDs and long-form packets for connections that have
-incomplete cryptographic handshakes are associated with an existing connection
-using the tuple of source and destination IP addresses and ports.
-
-A packet that uses the short header could be associated with an existing
-connection with an incomplete cryptographic handshake.  Such a packet could be a
-valid packet that has been reordered with respect to the long-form packets that
-will complete the cryptographic handshake.  This might happen after the final
-set of cryptographic handshake messages from either peer.  These packets are
-expected to be correlated with a connection using the tuple of IP addresses and
-ports.  Packets that might be reordered in this fashion SHOULD be buffered in
-anticipation of the handshake completing.
-
-0-RTT packets might be received prior to a Client Initial packet at a server.
-If the version of these packets is acceptable to the server, it MAY buffer these
-packets in anticipation of receiving a reordered Client Initial packet.
+with an existing connection, or - for servers - potentially create a new
+connection.
+
+Host handle packets that can be associated with an existing connection
+according to the current state of that connection. Short form packets without
+connection IDs, and long-form packets for connections that have incomplete
+cryptographic handshakes, are associated with an existing connection using the
+tuple of source and destination IP addresses and ports. Other packets are
+associated with existing connections using connection ID the connection ID
+in the header; this might include connection IDs that were advertised using
+NEW_CONNECTION_ID ({{frame-new-connection-id}}).
+
+Clients SHOULD discard any packet that cannot be associated with an existing
+connection.  Discarded packets MAY be logged for diagnostic or security
+purposes.
+
+If a server receives a packet not associated with an existing connection, it
+executes the following steps, in order:
+
+The server MUST check if the packet uses a short form header, or is not long
+enough for the size required for the initial packet of any QUIC version that
```

Comment:
Let's avoid using "initial" and instead use "first" here.  It could be confused with the packet type.